lib/Components/CollectionComponent.ts

```diff
@@ -85,17 +100,17 @@ export class CollectionComponent extends Component {
      *
      * @private
      */
-    _prepareArrayContent() {
-        return this._defaultPrepare(this.collection.map(model => this.prepareChild(model)));
+    protected _prepareArrayContent() {
+        return this._defaultPrepare((this.collection as Object[]).map(model => this.prepareChild(model)));
     }
 
     /**
      * @returns {Component[]}
      *
      * @private
      */
-    _prepareValueProxyContent() {
-        return this._defaultPrepare(this.collection.getValue().map(model => this.prepareChild(model)));
+    protected _prepareValueProxyContent() {
```

Comment:
should be renamed now to protected _prepareValueProxyContent()
protected + __ is tautology